experimental/packages/exporter-logs-otlp-http/README.md

```diff
@@ -0,0 +1,115 @@
+# OpenTelemetry Collector Logs Exporter for web and node with HTTP
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This module provides exporter for web and node to be used with OTLP (`http`) compatible receivers.
+Compatible with [opentelemetry-collector][opentelemetry-collector-url] versions `>=0.54.0`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/exporter-logs-otlp-http
+```
+
+## Service Name
+
+The OpenTelemetry Collector Exporter does not have a service name configuration.
+In order to set the service name, use the `service.name` resource attribute as prescribed in the [OpenTelemetry Resource Semantic Conventions][semconv-resource-service-name].
+To see documentation and sample code for the traces exporter, as well as instructions for using TLS, visit the [Collector Trace Exporter for web and node][trace-exporter-url].
+To see documentation and sample code for the metric exporter, see the [exporter-metrics-otlp-grpc package][metrics-exporter-url]
+
+## Logs in Web
+
+The OTLPLogExporter in Web expects the endpoint to end in `/v1/logs`.
+
+```js
+import { SeverityNumber } from '@opentelemetry/api-logs';
+import {
+  LoggerProvider,
+  BatchLogRecordProcessor,
+} from '@opentelemetry/sdk-logs';
+import { OTLPLogExporter } from '@opentelemetry/exporter-logs-otlp-http';
+
+const collectorOptions = {
+  url: '<opentelemetry-collector-url>', // url is optional and can be omitted - default is http://localhost:4318/v1/logs
+  headers: {}, // an optional object containing custom headers to be sent with each request
+  concurrencyLimit: 1, // an optional limit on pending requests
+};
+const logExporter = new OTLPLogExporter(collectorOptions);
+const loggerProvider = new LoggerProvider();
+
+loggerProvider.addLogRecordProcessor(new BatchLogRecordProcessor(logExporter));
+
+const logger = loggerProvider.getLogger('default', '1.0.0');
+// Emit a log
+logger.emit({
+  severityNumber: SeverityNumber.INFO,
+  severityText: 'info',
+  body: 'this is a log body',
+  attributes: { 'log.type': 'custom' },
+});
+```
+
+## Logs in Node
```

Comment:
Yes, it's almost the same here, mainly because the whole process is the same, provider -> processor -> exporter -> emit log, but the exporter part is different, one is `OTLPBrowserExporter` and the other is `OTLPNodeExporter`, and the corresponding configurations are also different, one is `OTLPExporterConfigBase` and the other is `OTLPExporterNodeConfigBase`. Writing them separately will make it easier to view and understand (I also added comments on the configuration in the example section:https://github.com/open-telemetry/opentelemetry-js/pull/3764/commits/4c51ed197169b3bf220cfa566245bc5fe7936176). The `trace` and `metric` parts are also written separately. Would it be better to write them separately here?